db/migrate/20170620222146_change_games_columnlast_move_pawn_two_steps_second_time.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeGamesColumnlastMovePawnTwoStepsSecondTime < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :games, :last_move_en_passant_situation?, :piece_capturable_by_en_passant
+    change_column :games, :piece_capturable_by_en_passant, :string
```

Comment:
Thanks for the feedback, before I wanted to store the whole piece class in the database but it turned out to be a bit difficult (tired it with a :string column but it would only store the first part "#<Pawn:0x007fae478a0b68>" and after some google searches I saw that it would be difficult so I decided just to store the id)